### ISSUE TITLE
Forward and backward buttons in system audio controller don't work

### ISSUE THREAD

**Issue by BlueManCZ** [martpie/museeks#525]
As title says, I can use only pause/play button. The rest is disabled.

![Snímek z 2020-03-27 15-51-36](https://user-images.githubusercontent.com/17854950/77769266-e2e50100-7043-11ea-8718-f2eb37f16a26.png)

This controller also manages inputs from the keyboard media keys, so full functionality would be nice.

**Comment by BlueManCZ** [comment]
And it also disappears for some unknown reason after some time. But pause/play on keyboard still works.

![Snímek z 2020-03-27 16-03-12](https://user-images.githubusercontent.com/17854950/77769808-9fd75d80-7044-11ea-91bb-bf20341fa96e.png)

**Comment by martpie** [comment]
can you maybe try the version of Museeks that is now built on Linux? It may fix the mpris issue

https://github.com/martpie/museeks/actions/runs/444618423 -> at the bottom: "binaries-linux"

**Comment by BlueManCZ** [comment]
It does basically the same. Still cannot switch tracks back or forward. I noticed I can switch one track with my forward/backward button on the keyboard, but only one track. Then it doesn't work anymore and I have to restart the application.

![obrazek](https://user-images.githubusercontent.com/17854950/103168551-6088a800-4834-11eb-8e33-972366ce447a.png)

I don't see any significant errors in the terminal.

```
$ ./museeks                   
[INFO] Skipping load of ThumbarModule
[INFO] Skipping load of DockMenuDarwinModule
[ERROR] An error occured when loading Object could not be loaded:
TypeError: Cannot read property 'indexOf' of undefined
(node:22718) electron: The default of contextIsolation is deprecated and will be changing from false to true in a future release of Electron.  See https://github.com/electron/electron/issues/23506 for more information
```

No errors in Developer Tools.

**Comment by martpie** [comment]
Thank you very much, so it seems the problem is with the library we use to do the mpris integration, and not the packaging system 👍

**Comment by BlueManCZ** [comment]
I can confirm this problem is fixed in [0.13.0](https://github.com/martpie/museeks/releases/tag/0.13.0), closing the issue.
